src/data/pyproject.toml

```diff
@@ -22,6 +22,8 @@ dependencies = [
     "scipy==1.10.1",
     "jsonlines==3.1.0",
     "datasketch<2.0.0",
-    "beautifulsoup4==4.12.2"
+    "beautifulsoup4==4.12.2",
+    "scikit-learn==1.2.2",
+    "hydra-core==1.1.0"
```

Comment:
```suggestion
    "hydra-core==1.1.0, < 2"
```

Reply:
Just saw that hydra-core is in `core/pyproject.toml` so I remove this line krub